content/actions/reference/workflow-syntax-for-github-actions.md

```diff
@@ -181,6 +181,25 @@ The filter determines if a workflow should run by evaluating the changed files a
 
 For more information, see "[About comparing branches in pull requests](/articles/about-comparing-branches-in-pull-requests)."
 
+## `on.workflow_dispatch.inputs`
+
+When using `workflow_dispatch` event, you can optionally specify inputs, which will be presented as form elements in the UI. Workflow dispatch inputs are specified with the same format as action inputs. For more information about format see "[Metadata syntax for GitHub Actions](/actions/creating-actions/metadata-syntax-for-github-actions#inputs)."
```

Comment:
Since you can execute workflows outside of the UI (e.g. through the GitHub CLI), I would recommend omitting specific reference to the UI:
```suggestion
When using `workflow_dispatch` event, you can optionally specify inputs that are passed to the workflow. Workflow dispatch inputs are specified with the same format as action inputs. For more information about the format see "[Metadata syntax for GitHub Actions](/actions/creating-actions/metadata-syntax-for-github-actions#inputs)."
```